flocx_market/tests/unit/common/test_service.py

```diff
@@ -39,12 +39,12 @@ def test_service_cli_valid(find_config_files):
            str(args[args.index('--api-public_endpoint') + 1]))
     assert(CONF.api.api_workers ==
            int(args[args.index('--api-api_workers') + 1]))
-    assert(CONF.api.enable_ssl_api)
+    assert CONF.api.enable_ssl_api
 
     assert(CONF.api.max_limit == 1000)
 
     assert(CONF.flask.SQLALCHEMY_TRACK_MODIFICATIONS is False)
-    assert(CONF.flask.PROPAGATE_EXCEPTIONS)
+    assert CONF.flask.PROPAGATE_EXCEPTIONS
```

Comment:
There's some inconsistent use of parentheses here.  If we don't need them to extend the `assert` statement across multiple lines, then we should drop the parentheses.  In other words: I agree with your change, but it looks like we should make the same change on additional lines.

Reply:
The only reason I remove the parenthesis for these 2 is that flake8 is failing if I leave them, but for the others, we need the parenthesis, we only need to remove them for the single expressions ones @larsks

Reply:
I've updated the PR to include more consistent formatting for the `assert` lines.